GameProject/Assets/OpenWorld Assets/Scripts/ThirdPersonCharacterController.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ThirdPersonCharacterController : MonoBehaviour
```

Comment:
This class is currently in the global:: namespace. Please provide a proper namespace.